frontend/integration-tests-cypress/cypress-e2e/cypress/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Do we need a separate ts config or can the root one apply?

Reply:
Sorry, Typescript configuration isn't my strong suit.   I do know when I tried to use the root one our existing Protractor tests wouldn't compile because they were using the Cypress's version of Chi or Mocha.

Reply:
I created a separate dev branch and attempted to add `cypress` and `@cypress/webpack-preprocessor` to `frontend/package.json`, but got many conflicts/errors.

Reply:
Still need separate `tsconfig.json` file.
See [comment](https://github.com/openshift/console/pull/4831#issuecomment-611705992)